var/spack/repos/builtin/packages/py-fastremap/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyFastremap(PythonPackage):
+    """Renumber and relabel Numpy arrays at C++ speed and physically convert rectangular
+    Numpy arrays between C and Fortran order using an in-place transposition"""
+
+    homepage = "https://github.com/seung-lab/fastremap/"
+    pypi = "fastremap/fastremap-1.2.1.tar.gz"
```

Comment:
```suggestion
    pypi = "fastremap/fastremap-1.14.1.tar.gz"
```
Doesn't actually matter for downloading but is occasionally used in `spack url summary`